src/hooks/common/useToast.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef, useState } from 'react';
+
+import { TOAST_SHOW_DURATION } from '../../constants/ui';
+
+const useToast = () => {
+  const [isAdded, setIsAdded] = useState(false);
```

Comment:
```suggestion
const useToast = (initIsAdd = false) => {
  const [isAdded, setIsAdded] = useState(initIsAdd);
```
이런 형태로 확장성을 열어둘 수 있지 않을까요?
아! 물론 일부러 숨기셨다면 그것도 괜찮네요 ㅎㅎ

Reply:
일부로 숨겼습니다! Toast가 특성상(?) 처음부터 보이는 상황은 없다고 생각했기 때문입니다.

Reply:
오홍 네네 좋습니다